src/components/block/index.ts

```diff
@@ -367,8 +367,8 @@ export default class Block {
    *
    * @returns {boolean}
    */
-  public get isEmpty(): boolean {
-    const emptyText = $.isEmpty(this.pluginsContent);
+  public isEmpty(ignoreSpaces: boolean): boolean {
```

Comment:
I think getter/setter should not be used for the future maintainability. 
See: https://github.com/airbnb/javascript#accessors--no-getters-setters